TLM/TLM/Lifecycle/TMPELifecycle.cs

```diff
@@ -36,18 +67,13 @@ public class LoadingExtension : LoadingExtensionBase {
         /// </summary>
         internal static bool PlayMode => AppMode != null && AppMode == ICities.AppMode.Game;
 
-        /// <summary>
-        /// Contains loaded languages and lookup functions for text translations
-        /// </summary>
-        public static Translation TranslationDatabase = new Translation();
-
-        public static List<ICustomManager> RegisteredManagers { get; private set; }
-
-        public static bool IsGameLoaded { get; private set; }
 
-        private static void RegisterCustomManagers() {
-            RegisteredManagers = new List<ICustomManager>();
+        private static void CheckForIncompatibleMods() {
+            ModsCompatibilityChecker mcc = new ModsCompatibilityChecker();
+            mcc.PerformModCheck();
+        }
 
+        private void RegisterCustomManagers() {
```

Comment:
[Bug introduced]
List was "reset" (instantiated) before change. (first call is `TMPELifecycle.Awake()` then second one in the `TMPELifecycle.Load()`)
Ended up on multiple instances in the list -> multiple calls to `OnLevelLoading()` and `OnLevelUnloading()`

Reply:
thanks :)